src/AddressRegistry.Consumer.Read.StreetName/StreetNameConsumer.cs

```diff
@@ -0,0 +1,52 @@
+namespace AddressRegistry.Consumer.Read.StreetName
+{
+    using System.Threading;
+    using System.Threading.Tasks;
+    using Autofac;
+    using Be.Vlaanderen.Basisregisters.MessageHandling.Kafka.Simple;
+    using Be.Vlaanderen.Basisregisters.ProjectionHandling.Connector;
+    using Projections;
+
+    public class StreetNameConsumer
+    {
+        private readonly ILifetimeScope _container;
+        private readonly KafkaOptions _options;
+        private readonly StreetNameConsumerOptions _streetNameConsumerOptions;
+
+        public StreetNameConsumer(
+            ILifetimeScope container,
+            KafkaOptions options,
+            StreetNameConsumerOptions streetNameConsumerOptions)
+        {
+            _container = container;
+            _options = options;
+            _streetNameConsumerOptions = streetNameConsumerOptions;
+        }
+
+        public Task Start(CancellationToken cancellationToken = default)
+        {
+            var latestItemProjector = new ConnectedProjector<StreetNameConsumerContext>(Resolve.WhenEqualToHandlerMessageType(new StreetNameLatestItemProjections().Handlers));
+            var bosaItemProjector = new ConnectedProjector<StreetNameConsumerContext>(Resolve.WhenEqualToHandlerMessageType(new StreetNameBosaItemProjections().Handlers));
+
+            var consumerGroupId = $"{nameof(AddressRegistry)}.{nameof(StreetNameConsumer)}.{_streetNameConsumerOptions.Topic}{_streetNameConsumerOptions.ConsumerGroupSuffix}";
+            return KafkaConsumer.Consume(
+                new KafkaConsumerOptions(
+                    _options.BootstrapServers,
+                    _options.SaslUserName,
+                    _options.SaslPassword,
+                    consumerGroupId,
+                    _streetNameConsumerOptions.Topic,
+                    async message =>
+                    {
+                        var streetnameConsumerContext = _container.Resolve<StreetNameConsumerContext>();
+                        await latestItemProjector.ProjectAsync(streetnameConsumerContext, message, cancellationToken);
+                        await bosaItemProjector.ProjectAsync(streetnameConsumerContext, message, cancellationToken);
```

Comment:
Seeing this, we will have to split it up into 2 consumer tasks, each with its own suffix.
What if line 42 is succesful and 43 isn't? Consumer will see this as not processed, but if you reprocess it line 42 might crash.